configure

```diff
@@ -488,7 +492,8 @@ def pkg_config(pkg):
           shlex.split(pkg_config) + ['--silence-errors', flag, pkg],
           stdout=subprocess.PIPE)
       val = proc.communicate()[0].strip()
-    except OSError, e:
+    except OSError:
```

Comment:
Ah, to be exact, it will parse in >= 2.6 which is the minimum we support, so it's fine I think.